### PR TITLE
PLANET-5756 Remove jQuery from external_link.js and pdf_icon.js

### PR DESCRIPTION
**Pull request by mleray**
### Description
See https://jira.greenpeace.org/browse/PLANET-5756

I've also removed the link hover color to align with the design system and https://jira.greenpeace.org/browse/PLANET-5843

### Related PRs
- master-theme: https://github.com/greenpeace/planet4-master-theme/pull/1295

### CODE REVIEW

**Comment by comzeradd** [comment]
Looks like you have a redundant merge commit

**Comment by mleray** [comment]
> Looks like you have a redundant merge commit

I think it's solved now...?

File: src/base/_icons.scss [hunk 1]
```diff
@@ -32,21 +49,39 @@ a.pdf-link {
       mask-image: url("../../images/file-pdf.svg");
       mask-repeat: no-repeat;
       -webkit-mask-repeat: no-repeat;
-      margin-right: 0;
+      mask-position: center;
+      -webkit-mask-position: center;
```
Comment (Inwerpsel) [inline src/base/_icons.scss:53]: It looks like autoprefixer is already adding these `-webkit-` prefixed versions for us, at least for `mask-position` it does so locally for me. So we can probably remove them all.

Comment (mleray) [reply src/base/_icons.scss]: Updated! 🙂

**Review by Inwerpsel — APPROVED** [review]
:100:

**Pull request merged**